infrastructure/dogfood/terraform/aws-tf-module/main.tf

```diff
@@ -26,7 +26,7 @@ terraform {
 
 variable "fleet_license" {}
 variable "fleet_image" {
-  default = "160035666661.dkr.ecr.us-east-2.amazonaws.com/fleet:7408a0df90802fbd602b52015546dd46590051bd"
+  default = "160035666661.dkr.ecr.us-east-2.amazonaws.com/fleet:1f68e7a5e39339d763da26a0c8ae3e459b2e1f016538d7962312310493381f7c"
```

Comment:
As a note, with the job now working, we should expect this to change... a lot.

Reply:
Agreed and understood.  I just made it match live for the `terraform plan` from the CLI to match up and not try and change the task definition.  I figured it wasn't worth reverting the line.